docs/package-inclusion-guide.md

```diff
@@ -0,0 +1,55 @@
+# Azure Cloud Shell Package Inclusion Guide
+
+## Background
+
+There are millions (or potentially billions) of Linux packages. This guide goes through the principles of what the Azure Cloud Shell team uses to consider what gets included in the service.
+
+
+
+## General Principles
+
+1. Useful for Azure users and administrators 
+2. Available on [Azure Linux Operating System](https://github.com/microsoft/azurelinux) 
+3. Vulnerability free 
+    - No unfixed CVEs for the package on Azure Linux OS past 30 days 
+4. Appropriate Licensing 
+    - Azure Cloud Shell should be allowed to both consume and distribute the package 
+5. Well-maintained and supported 
+    - Package should NOT be deprecated or archived 
+    - Package repo should have a contribution/commit within the last 120 days 
+    - The package should be of a stable version. The package should not be in alpha or beta version. 
+
+6. Reasonable package size 
```

Comment:
```suggestion
6. Is reasonably sized:
    - Cloud Shell cannot accommodate something that is gigabyte in size.
```

Reply:
I personally like having it more generic